### PR TITLE
WIP: Trivial: Updated outdated GitHub urls

### PR DESCRIPTION
**Pull request by gto90**
Updated old GitHub repository URLs with the DigiByte-Core url.

### CODE REVIEW

File: contrib/debian/changelog [hunk 1]
```diff
@@ -36,7 +36,7 @@ digibyte (0.13.2-trusty1) trusty; urgency=medium
 
 digibyte (0.13.1-trusty2) trusty; urgency=medium
 
-  * Revert to Qt4, due to https://github.com/digibyte/digibyte/issues/9038
+  * Revert to Qt4, due to https://github.com/DigiByte-Core/digibyte/issues/9038
```
Comment (ChillingSilence) [inline contrib/debian/changelog:39]: Some such as this are broken still, as these should be github.com/bitcoin/bitcoin/issues/9038

File: contrib/gitian-descriptors/gitian-win.yml [hunk 2]
```diff
@@ -74,7 +74,7 @@ script: |
 
   function create_per-host_linker_wrapper {
   # This is only needed for trusty, as the mingw linker leaks a few bytes of
-  # heap, causing non-determinism. See discussion in https://github.com/digibyte/digibyte/pull/6900
+  # heap, causing non-determinism. See discussion in https://github.com/DigiByte-Core/digibyte/pull/6900
```
Comment (ChillingSilence) [inline contrib/gitian-descriptors/gitian-win.yml:77]: Another one for bitcoin repos

File: contrib/rpm/README.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ reference the release tarball available from https://digibyte.org/bin/ and
 `Source1` should reference the BerkeleyDB source.
 
 Sources 10-99 are for source files that are maintained in the
-[DigiByte git repository](https://github.com/digibyte/digibyte) but are not part of
+[DigiByte git repository](https://github.com/DigiByte-Core/digibyte) but are not part of
```
Comment (ChillingSilence) [inline contrib/rpm/README.md:22]: Should probs even just look at removing this file / directory, given we don't maintain any rpms and this is out of date

File: depends/README.md [hunk 4]
```diff
@@ -49,7 +49,7 @@ For linux RISC-V 64-bit cross compilation (there are no packages for 32-bit):
 
     sudo apt-get install curl g++-riscv64-linux-gnu binutils-riscv64-linux-gnu
 
-RISC-V known issue: gcc-7.3.0 and gcc-7.3.1 result in a broken `test_digibyte` executable (see https://github.com/digibyte/digibyte/pull/13543),
+RISC-V known issue: gcc-7.3.0 and gcc-7.3.1 result in a broken `test_digibyte` executable (see https://github.com/DigiByte-Core/digibyte/pull/13543),
```
Comment (ChillingSilence) [inline depends/README.md:52]: For bitcoin/bitcoin

File: doc/REST-interface.md [hunk 5]
```diff
@@ -54,7 +54,7 @@ Only supports JSON as output format.
 
 The getutxo command allows querying of the UTXO set given a set of outpoints.
 See BIP64 for input and output serialisation:
-https://github.com/digibyte/bips/blob/master/bip-0064.mediawiki
+https://github.com/DigiByte-Core/bips/blob/master/bip-0064.mediawiki
```
Comment (ChillingSilence) [inline doc/REST-interface.md:57]: Also for bitcoin/bitcoin

**Review by ChillingSilence — CHANGES_REQUESTED** [review]
There's a bunch that have been changed to digibyte-core/digibyte which should be bitcoin/bitcoin and a few things that are just not relevant to us at all which we should just nuke.

**Comment by gto90** [comment]
Good catch @ChillingSilence , I'll work through those and update accordingly.

**Comment by ChillingSilence** [comment]
Good stuff mate , and ty for getting these fixed 👍

**Comment by gto90** [comment]
I am going to close this and reopen it against the feature branch.

**Pull request closed without merging**